python/lsst/ts/salobj/topics/controller_command.py

```diff
@@ -279,6 +279,4 @@ def _setup(self):
         self._DataType = getattr(self.salinfo.lib, self.salinfo.name + "_command_" + self.name + "C")
 
         topic_name = self.salinfo.name + "_command_" + self.name
-        retcode = self.salinfo.manager.salProcessor(topic_name)
-        if retcode != self.salinfo.lib.SAL__OK:
-            raise RuntimeError(f"salProcessor({topic_name}) failed with return code {retcode}")
+        self.salinfo.manager.salProcessor(topic_name)
```

Comment:
Do you think you can catch the exception and then trow it again but printing which topic failed? the current message obscures the failed topic so we need to go look at the log file to figure out. Either this of do it inside SAL.

Reply:
Yes. It would be better if SAL included the information itself, but that may be awhile before that happens so I have added it here.